addons/tmf_loadouts/loadouts/ru_army_template_2020.hpp

```diff
@@ -555,13 +555,15 @@ class sp : g {
 	};
 };
 
-class vg : smg {
+class vg : car {
```

Comment:
If you're doing this instead give them an AKS-74U, besides weren't you the one that begged me to give them a vikhr?

Reply:
Turns out that I was wrong xD

Reply:
Also, apparently, the AK105 has replaced the AKS-74U due to unresolved reliability issues with the short gas block and booster, so the 105 is issued instead.

Reply:
Actually, just looked back at the changelog and it looks like you were gonna give them the PP-19-01 Vityaz, which is even less kosher xDD